manifests/user.pp

```diff
@@ -13,9 +13,10 @@
   }
 
   ssh_authorized_key { "${username}_default":
-    user => $username,
-    type => $pubkey_type,
-    key  => $pubkey,
+    user   => $username,
+    type   => $pubkey_type,
+    key    => $pubkey,
+    target => "${home}/.ssh/authorized_keys2",
```

Comment:
No -- that's already the default behavior on Debian and Ubuntu, so no additional change is needed. I verified that I can authenticate with keys placed in either one, so in theory we don't even need this; I think it's a good idea to put the managed keys in the alternate location, though, because of tools like ssh-copy-id that assume the user wants to install in ~/.ssh/authorized_keys.